### ISSUE TITLE
Add Bits Denomination by default (BIP176)

### ISSUE THREAD

**Issue by thinkjanis** [LN-Zap/zap-desktop#137]
Since there will be a lot of interest in using Zap with smaller transactions, this would be a great time to introduce Bits Denomination. For most users, it would be easier to switch from fiat because bits allow displaying the full Bitcoin balance with 2 decimal places and the same format is used with traditional currencies.

A more detailed explanation of the benefits by @jimmysong
https://github.com/bitcoin/bips/blob/master/bip-0176.mediawiki

**Comment by ChuckSRQ** [comment]
I disagree with having Bits as default. I personally do not like it.  Users should be given the option of which to choose.

**Comment by thinkjanis** [comment]
No one will force you to use it @ChuckSRQ. The idea is to have multiple options and Bits by default will be beneficial for various reasons explained above.
